packages/flutter_tools/lib/src/base/process.dart

```diff
@@ -10,6 +10,8 @@ import '../globals.dart';
 
 typedef String StringConverter(String string);
 
+final Map<String, String> childEnvironmentOverrides = new Map<String, String>();
```

Comment:
Reading this: https://api.dartlang.org/stable/1.16.1/dart-io/Process/run.html, it looks like it defaults to just overridding the specified vars, so looks ok.